package-lock.json

```diff
@@ -4192,7 +4192,8 @@
       "version": "1.3.0",
       "resolved": "https://registry.npmjs.org/growly/-/growly-1.3.0.tgz",
       "integrity": "sha1-8QdIy+dq+WS3yWyTxrzCivEgwIE=",
-      "dev": true
+      "dev": true,
+      "optional": true
```

Comment:
Are the `package-lock.json` modifications coming with the change of `.eslintrc.js`?

Reply:
I think it's from a local `npm install` of mine. I can remove it from the PR if anyone thinks it's causing harm.

Reply:
https://stackoverflow.com/a/55373854

Reply:
I just would like to avoid adding it here then remove it somewhere else. That can happen easily if the team does not use the same npm.

Reply:
I don't think the changes in there cause any harm, but I've removed it from the PR to move it along.